### PR TITLE
Hyperlink extract from office docs - added to extract indicators from file playbook

### PR DESCRIPTION
**Pull request by ssokolovich**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-9869)

## Description
Added an option to extract hyper-links from office files.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by tomer-pan — APPROVED** [review]

File: Packs/CommonPlaybooks/Playbooks/playbook-Extract_Indicators_From_File_-_Generic_v2_4_5.yml [hunk 1]
```diff
@@ -1893,6 +1912,185 @@ tasks:
...
     isautoswitchedtoquietmode: false
+  "26":
+    id: "26"
+    taskid: 608bdaf3-5ea5-4ea7-85bc-0343f8535b0d
+    type: condition
+    task:
+      id: 608bdaf3-5ea5-4ea7-85bc-0343f8535b0d
+      version: -1
+      name: Is there a PPTX/XSLX file?
+      description: Checks if there is a Word file (DOC, DOCX) in context.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Extract_Indicators_From_File_-_Generic_v2_4_5.yml:1923]: The name and description do not match.
The name says it is looking for a **Powerpoint** or **Excel** file.
The description asks if a **Word** file was found.
Confusing.

File: Packs/CommonPlaybooks/ReleaseNotes/2_6_27.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Extract Indicators From File - Generic v2
+
+- Added the ability to extract hyper-links from office files (supporting .xlsx,.pptx,.docx).
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/ReleaseNotes/2_6_27.md:6]:
```suggestion
Added the ability to extract hyperlinks from Office files (supporting .xlsx,.pptx,.docx).
```

File: Packs/CommonPlaybooks/Playbooks/playbook-Extract_Indicators_From_File_-_Generic_v2_4_5.yml [hunk 3]
```diff
@@ -2175,9 +2373,19 @@ outputs:
 - contextPath: QRCodeReader.IP
   description: The IPs extracted from the QR code image if they are present.
   type: String
+- contextPath: ExtractedHyperLink
+  description: Extracted Hyperlinks from supported office documents (xlsx,docx,pptx).
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Extract_Indicators_From_File_-_Generic_v2_4_5.yml:2377]:
```suggestion
  description: Extracted hyperlinks from supported Office documents (xlsx,docx,pptx).
```

File: Packs/CommonPlaybooks/Playbooks/playbook-Extract_Indicators_From_File_-_Generic_v2_4_5.yml [hunk 4]
```diff
@@ -1893,6 +1912,185 @@ tasks:
...
+    isautoswitchedtoquietmode: false
+  "27":
+    id: "27"
+    taskid: eefbac0c-db14-4ba0-85f7-fbb67a0f40b3
+    type: regular
+    task:
+      id: eefbac0c-db14-4ba0-85f7-fbb67a0f40b3
+      version: -1
+      name: Extract Hyper-Links from Office Files
+      description: 'Extracts hyperlinks from office files. Supported file types are: xlsx, docx, pptx.'
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Extract_Indicators_From_File_-_Generic_v2_4_5.yml:2016]:
```suggestion
      description: 'Extracts hyperlinks from Office files. Supported file types are: xlsx, docx, pptx.'
```

File: Packs/CommonPlaybooks/Playbooks/playbook-Extract_Indicators_From_File_-_Generic_v2_4_5.yml [hunk 5]
```diff
@@ -2175,9 +2373,19 @@ outputs:
...
   description: The IPs extracted from the QR code image if they are present.
   type: String
+- contextPath: ExtractedHyperLink
+  description: Extracted Hyperlinks from supported office documents (xlsx,docx,pptx).
+  type: unknown
+- contextPath: ExtractedHyperLink.URL
+  description: The URL of the extracted hyperlink.
+  type: String
+- contextPath: ExtractedHyperLink.FileName
+  description: The office file from which the hyperlinks are extracted.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Extract_Indicators_From_File_-_Generic_v2_4_5.yml:2383]:
```suggestion
  description: The Office file from which the hyperlinks are extracted.
```

**Comment by ShirleyDenkberg** [comment]
@tomer-pan @idovandijk Doc review completed.

File: Packs/CommonPlaybooks/Playbooks/playbook-Extract_Indicators_From_File_-_Generic_v2_4_5.yml (hunk 1 above)
Comment (ssokolovich) [inline Packs/CommonPlaybooks/Playbooks/playbook-Extract_Indicators_From_File_-_Generic_v2_4_5.yml:1923]:
```suggestion
      description: Checks if there is a PowerPoint or Excel file (PPTX/XSLX) in context.
```

Comment (ssokolovich) [reply Packs/CommonPlaybooks/Playbooks/playbook-Extract_Indicators_From_File_-_Generic_v2_4_5.yml:1923]: agree, missed that. TNX

**Pull request merged**